crates/wash-lib/src/build.rs

```diff
@@ -126,6 +126,17 @@ pub fn build_actor(
                 };
                 actor_wasm_path
             }
+            LanguageConfig::Other(_) if actor_config.build_command.is_some() => {
+                // SAFETY: We checked that the build command is not None above
```

Comment:
NIT: don't think you need this comment

Reply:
I figure it doesn't hurt, I just like including them wherever there is an `unwrap` in library code. I can remove if it bugs ya 😉

Reply:
oh yeah definitely not a big deal! Resolved!